### PR TITLE
Improved the gradle system and the code style

### PR DESCRIPTION
**Pull request by liach**
I improved the code of minigamecore to match sponge code style somehow and added a few more gradle tools.

### CODE REVIEW

**Comment by Flibio** [comment]
Everything looks good except for the comments I mentioned above. I already had the formatting changes on my local drive, so I just pushed them to master.

**Comment by liach** [comment]
See https://jitpack.io/#liachmodded/MinigameCore
It'd be a great idea to grab minigamecore from maven instead of adding as git submodules.

File: build.gradle [hunk 1]
```diff
@@ -35,10 +26,21 @@ repositories {
         name 'Sponge API repo'
         url 'http://repo.spongepowered.org/maven'
     }
+    maven {
+        name 'jitpack'
+        url 'https://jitpack.io'
+    }
```
Comment (Flibio) [inline build.gradle]: I don't think you quite understand. JitPack does not need to be added as a repository for it to publish the plugin. It only needs to be added if you want to use a published plugin.

**Comment by Flibio** [comment]
@liach 

Are you referencing Ultimate Spleef? If so, that was something I was just trying.

File: src/main/java/me/flibio/minigamecore/Minigame.java [hunk 2]
```diff
@@ -30,13 +30,12 @@
...
 import me.flibio.minigamecore.scoreboards.ScoreboardManager;
 import me.flibio.minigamecore.teams.TeamManager;
-
 import org.slf4j.Logger;
 import org.spongepowered.api.Game;
 
 public class Minigame {
 
-    // Managers
+    //Managers
```
Comment (Flibio) [inline src/main/java/me/flibio/minigamecore/Minigame.java]: The whitespace here is part of the Sponge and Google code style:

Quote from https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace, number 6 under `4.6.2 Horizontal whitespace`: 

> Beyond where required by the language or other style rules, and apart from literals, comments and Javadoc, a single ASCII space also appears in the following places only.
> ...
> 6. On both sides of the double slash (`//`) that begins an end-of-line comment.

**Comment by Flibio** [comment]
Closing as MinigameCore is going to be reworked soon. These changes will be kept in mind.

**Comment by liach** [comment]
Okay.

**Pull request closed without merging**